tests/forms_tests/tests/test_forms.py

```diff
@@ -2377,3 +2377,53 @@ def test_baseform_repr_dont_trigger_validation(self):
         self.assertRaises(AttributeError, lambda: p.cleaned_data)
         self.assertFalse(p.is_valid())
         self.assertEqual(p.cleaned_data, {'first_name': 'John', 'last_name': 'Lennon'})
+
+    def test_field_name_with_hidden_input(self):
+        """
+        BaseForm._html_output() uses inconsistent formatting for normal row
```

Comment:
I think the test name is good enough, so I'd remove the docstring or replace it with a more informational one.